EC-84: Restructure eZ Commerce doc, part 2 (#2)
ezsystems/commerce-documentation: docs/cookbook/frontend_customizing/4_front_end_stack_in_details/4.2_customized_foundation_framework/4.2._customized_foundation_framework.md

php bin/console assetic:dump
```

> If you need some features from the higher eZ Commerce version, you will need to create the templates by yourself.

Reviewer: I'd go along the lines of:
If you need features from the newer version of eZ Commerce, you must create the templates by yourself.
This way we avoid a repetition of "need". Also, when I look at a broader picture, I'd add another sentence that tells the users how they could create the templates.
For more information about adding templates, see [Adding tempates](https://doc.ezplatform.com/en/master/tutorials/extending_admin_ui/4_adding_a_custom_tag/#create-a-template).
If the table below lists available templates, I'd add a sentence that introduces the table, something like:
For a list of available templates, see the table below.
If the sass files is something different, I's add a sentence or two of an explanation.

> If you need some features from the higher eZ Commerce version, you will need to create the templates by yourself.

Reviewer 2: ```suggestion
If you need features from the newer version of eZ Commerce, you must create the templates by yourself.
```
